extensions/lifecycle/objectProcessor/policy.json

```diff
@@ -9,7 +9,8 @@
                 "s3:GetObjectVersion",
                 "s3:DeleteObject",
                 "s3:DeleteObjectVersion",
-                "s3:AbortMultipartUpload"
+                "s3:AbortMultipartUpload",
+                "s3:ReplicateObject"
```

Comment:
I am not sure to understand what lifecycle expiration "call" is "s3:ReplicateObject" action allowing.

Reply:
`s3:ReplicateObject` is the permission for all of the `backbeatRoute`